cloud/amazon/iam_policy.py

```diff
@@ -283,7 +283,7 @@ def main():
       iam_name=dict(default=None, required=False),
       policy_name=dict(default=None, required=True),
       policy_document=dict(default=None, required=False),
-      policy_json=dict(default=None, required=False),
+      policy_json=dict(type='dict', default=None, required=False),
```

Comment:
@bcoca Thanks.  Yeah, it makes sense.  It just isn't working on my machine with ansible 2.2.